src/main/java/org/gridsuite/study/server/dto/elasticsearch/EquipmentInfos.java

```diff
@@ -49,6 +49,9 @@ public class EquipmentInfos extends BasicEquipmentInfos {
     @Field("equipmentType")
     String type;
 
+    @Field(type = FieldType.Keyword)
+    Set<String> equipmentSubType;
```

Comment:
```suggestion
    Set<String> equipmentSubTypes;
```
Since it is a collection, should add 's' to the field name if the name has a same semantics